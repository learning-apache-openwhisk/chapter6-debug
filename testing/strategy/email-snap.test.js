const main = require("./email").main

test("validate email with snapshot", () => {
    expect(main({email: "michele.sciabarra@gmail.com"})).toMatchSnapshot()
    expect(main({email: "michele.sciabarra.com"})).toMatchSnapshot()
})
