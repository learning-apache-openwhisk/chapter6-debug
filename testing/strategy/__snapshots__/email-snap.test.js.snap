// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate email with snapshot 1`] = `
Object {
  "email": "michele.sciabarra@gmail.com",
  "errors": Array [],
  "message": Array [
    "email: michele.sciabarra@gmail.com",
  ],
}
`;

